crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -1574,393 +1415,6 @@ mod tests {
         stage.run(&mut world);
     }
 
-    #[test]
```

Comment:
This test is very fragile and hard to use due to the elaborate "label" reporting strategy.

I've done my best to replace the functionality over in the tests in `ambiguity_detection.rs`.